### PR TITLE
Ds4 73 as peter josephine i want to see a card for the assembly

### PR DESCRIPTION
**Pull request by GustavFrandsen**
image

### CODE REVIEW

File: backend/CommonAssembly/src/main/java/dk/sdu/controllab/commonassembly/GetInventoryAndStatusResponse.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import lombok.Setter;
+
+import java.util.List;
+//this class is for easier data transfer to the frontend
+@Getter
+@Setter
+public class GetInventoryAndStatusResponse {
+    private List<AssemblyInventoryEntry> inventory;
+    private GetStatusResponse statusResponse;
+    //private Recipe recipe;
```
Comment (Siole) [inline backend/CommonAssembly/src/main/java/dk/sdu/controllab/commonassembly/GetInventoryAndStatusResponse.java:14]: Should this comment be deleted?

File: backend/Recipe/src/main/java/dk/sdu/controllab/recipe/service/RecipeServiceImpl.java [hunk 2]
```diff
@@ -41,7 +41,9 @@ public List<Recipe> listRecipes() {
      */
     @Override
     public Recipe getRecipe(UUID recipeId) throws NoSuchElementException {
```
Comment (Siole) [inline backend/Recipe/src/main/java/dk/sdu/controllab/recipe/service/RecipeServiceImpl.java:43]: Redundant exception-handling

**Review by Siole — CHANGES_REQUESTED** [review]
A couple of minor changes to consider. 
In general we should look into the order of the cards and the order of the imports of cards in the dashboard (a team thing - not something you can currently fix)

File: backend/Assembly/src/main/java/dk/sdu/controllab/assembly/service/AssemblyServiceImpl.java [hunk 3]
```diff
@@ -30,9 +30,13 @@ public class AssemblyServiceImpl implements AssemblyService {
     private boolean isHealthy;
 
     private GetStatusResponse latestStatus;
+    //used to get the current recipeid
+    //@Value("00000000-0000-0000-0000-000000000000")
+   // private UUID latestRecipeId;
```
Comment (MagnusHJensen) [inline backend/Assembly/src/main/java/dk/sdu/controllab/assembly/service/AssemblyServiceImpl.java:35]: Remove all comments

**Review by 3FCO — APPROVED** [review]
Ser ud til at virke! 👍

**Pull request merged**